stdlib/public/Platform/Platform.swift

```diff
@@ -351,11 +351,6 @@ public func ioctl(
 public func fork() -> Int32 {
   fatalError("unavailable function can't be called")
 }
-
-@available(*, unavailable, message: "Please use threads or posix_spawn*()")
-public func vfork() -> Int32 {
```

Comment:
True, and I wondered about the same. I could undo this change and leave this message in, if you'd prefer.